packages/components/ng-at-internet/src/provider.js

```diff
@@ -18,8 +18,6 @@ import OvhAtInternet from '@ovh-ux/ovh-at-internet';
  */
 export default class NgAtInternet extends OvhAtInternet {
   $get() {
-    this.init();
```

Comment:
are we sure this doesn't break the component ?

Reply:
Actually, it does, and it's removed, it should never have been here. I did it in a previous PR and it was a mistake, so i'm removing it back to normal.